### PR TITLE
Workflow updates, fix nullable warnings

### PR DESCRIPTION
**Pull request by emanuelgaspar**

### CODE REVIEW

File: src/Trakx.Kaiko.ApiClient/Registration/KaikoClientRegistration.cs [hunk 1]
```diff
@@ -9,7 +9,10 @@ public static class KaikoClientRegistration
     public static IServiceCollection AddKaikoClient(this IServiceCollection services, IConfiguration configuration)
     {
         var config = configuration.GetSection(nameof(KaikoApiConfiguration)).Get<KaikoApiConfiguration>();
-        services.AddKaikoClient(config);
+        if (config != null)
```
Comment (joaofbvieira) [inline src/Trakx.Kaiko.ApiClient/Registration/KaikoClientRegistration.cs:12]: Shouldn't this throw an exception if the config doesn't exist?

Comment (emanuelgaspar) [reply src/Trakx.Kaiko.ApiClient/Registration/KaikoClientRegistration.cs:12]: We don't have that behaviour in other api clients. In other clients, we just assume the config is there. Here I merely tried to silence a warning. I can revert if it's easier to pass the PR 😅

Comment (emanuelgaspar) [reply src/Trakx.Kaiko.ApiClient/Registration/KaikoClientRegistration.cs:12]: <https://github.com/trakx/coingecko-api-client/blob/dev/src/Trakx.CoinGecko.ApiClient/AddCoinGeckoClientExtension.cs#L20>

Comment (emanuelgaspar) [reply src/Trakx.Kaiko.ApiClient/Registration/KaikoClientRegistration.cs:12]: <https://github.com/trakx/cryptocompare-api-client/blob/dev/src/Trakx.CryptoCompare.ApiClient/ServiceConfiguration.cs>

(examples of what I mean)

File: src/Trakx.Kaiko.ApiClient/Registration/KaikoClientRegistration.cs [hunk 2]
```diff
@@ -9,7 +9,10 @@ public static class KaikoClientRegistration
...
+        {
+            services.AddKaikoClient(config);
+        }
```
Comment (emanuelgaspar) [inline src/Trakx.Kaiko.ApiClient/Registration/KaikoClientRegistration.cs:15]:
```suggestion
        services.AddKaikoClient(config!);
```

File: src/Trakx.Kaiko.ApiClient/Registration/KaikoClientRegistration.cs (hunk 1 above)
Comment (emanuelgaspar) [reply src/Trakx.Kaiko.ApiClient/Registration/KaikoClientRegistration.cs]: It's fixed, `!` and done :)

**Review by joaofbvieira — APPROVED** [review]
LGTM =)

File: src/Trakx.Kaiko.ApiClient/Registration/KaikoClientRegistration.cs (hunk 1 above)
Comment (joaofbvieira) [reply src/Trakx.Kaiko.ApiClient/Registration/KaikoClientRegistration.cs]: Nice :)

**Pull request merged**